doc/source/index.rst

```diff
@@ -17,7 +17,6 @@ NumPy documentation
 **Version**: |version|
 
 **Download documentation**:
```

Comment:
I think the "Download documentation" text is not needed anymore. What do you think? [this](https://output.circle-artifacts.com/output/job/df62f9e6-64b7-4c60-a4d4-9e49a3e83d0b/artifacts/0/doc/build/html/index.html) is what it looks like after rendering.
```suggestion
```